config/config.go

```diff
@@ -1047,8 +1046,6 @@ func (cfg *BlockSyncConfig) ValidateBasic() error {
 	switch cfg.Version {
```

Comment:
I think we should remove `Version` entirely and just have `Enable` (and I'm not even sure we want to give the option to disable blocksync but we can think about that later).